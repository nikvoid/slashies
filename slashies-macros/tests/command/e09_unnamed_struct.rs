use slashies_macros::Command;

/// A command with no name
#[derive(Command)]
struct BadCommand;

fn main() {}
