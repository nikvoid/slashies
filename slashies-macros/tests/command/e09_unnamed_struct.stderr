error: Command must specify a name via the "name" attribute
 --> tests/command/e09_unnamed_struct.rs:5:8
  |
5 | struct BadCommand;
  |        ^^^^^^^^^^
